.syncpackrc

```diff
@@ -1,6 +1,6 @@
 {
 	"dev": true,
-	"filter": "^(?:react|react-dom|typescript)$",
+	"filter": "^(?:react|react-dom|typescript|@typescript-eslint|@types/react).*$",
```

Comment:
This filter wasn't working before, so the `@typescript-eslint/**` dependency group wasn't being version matched.

I find the filter kind of a weird API, It'd be nice if you could just have syncpack care about dependency matchers that you add in the `versionGroups` property, but I discovered it doesn't work that way.

Reply:
Can you open an issue @samueljseay please? Would be happy to take a look

Reply:
@JamieMason Sure I'll try and come up with a coherent suggestion 😄

Reply:
@samueljseay seeing this late, but I noticed this while looking into something else with syncpack. It looks like this change would also match `react.*`, etc (which makes `react-dom` redundant, for example).

What do you think about moving the .* into the group where it's needed? For example:
```suggestion
	"filter": "^(?:react|react-dom|typescript|@typescript-eslint.*|@types/react)$",
```

Reply:
I've not tested this but the following config should hopefully achieve what you want, it looks like you only want to check the versions of those listed dependencies, and no others?

```json
{
  "versionGroups": [
    {
      "dependencies": ["@types/react", "@typescript-eslint*", "react-dom", "react", "typescript"],
      "packages": ["**"]
    },
    {
      "dependencies": ["**"],
      "packages": ["**"],
      "isIgnored": true
    }
  ]
}
```

Notice the `*` after `"@typescript-eslint*"` to match anything that starts with `@typescript-eslint`

Reply:
I forgot to mention that with this in place you can delete the `filter` property as it won't be needed. I put [a note in the readme about filter](https://github.com/JamieMason/syncpack/#filter) but that was only fairly recently, I would avoid using it if you can.